src/igvfd/schemas/variant.json

```diff
@@ -0,0 +1,138 @@
+                "linkTo": "PhenotypeTerm",
+                "type": "string"
+            }
+        },
+        "dbxrefs": {
+            "title": "External Resources",
+            "description": "Unique identifiers from external resources.",
+            "comment": "This list should have at least an ID from the official nomenclature. The list of other dbxrefs is maintained using information from NCBI Entrez. Identifiers not included in NCBI gene_info can be added but are not actively maintained.",
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "title": "External identifier",
+                "description": "A unique identifier from external resource (e.g. HGNC, MGI, FlyBase, WormBase, ENSEMBL, MIM, UniProtKB, Vega, miRBase).",
+                "comment": "Submit as database_name:id. See changelog for potential databases.",
+                "type": "string",
+                "pattern": "^((HGNC:\\d+)|(MGI:\\d+)|(ENSEMBL:ENS[A-Z]*G\\d+)|(MIM:\\d+)|(UniProtKB:[0-9A-Z]+)|(Vega:OTT[A-Z]+G\\d+)|(miRBase:MI\\d+)|(IMGT/GENE-DB:[0-9A-Za-z/\\(\\)\\-]+)|(RefSeq:[A-Z]{2}_[\\d.]+)|(GeneCards:[0-9a-zA-Z-]+))$"
+            },
+            "permission": "import_items"
+        },
+        "variation_type": {
+            "title": "Variation Type",
+            "description": "The type of variant (SNV, duplication, insertion, duplication).",
+            "type": "string",
+            "enum": [
+                "SNV",
+                "insertion",
+                "deletion",
+                "duplication"
+            ]
+        },
+        "locations": {
+            "title": "Variant Locations",
+            "description": "Variant locations specified using 1-based, closed coordinates for different versions of reference genome assemblies.",
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "title": "Variant Location",
+                "description": "Variant location specified using 1-based, closed coordinates for a specific version of the reference genome assembly.",
+                "type": "object",
+                "additionalProperties": false,
+                "required": [
+                    "assembly",
+                    "chromosome",
+                    "position"
+                ],
+                "properties": {
+                    "assembly": {
+                        "title": "Mapping Assembly",
+                        "description": "The genome assembly to which coordinates relate. e.g. GRCh38.",
+                        "type": "string",
+                        "enum": [
+                            "GRCh38",
+                            "hg19",
+                            "mm10",
+                            "mm9"
+                        ]
+                    },
+                    "chromosome": {
```

Comment:
I think we should lift chromosome up to the main level as well.  variants do change position but they don't jump chromosomes.

Reply:
however, I think we have a few other things to talk about as a group before committing to that issue